casts/cards/src/UICollectionCard/UICollectionCard.tsx

```diff
@@ -73,7 +73,9 @@ export function UICollectionCard({
                             </UILabel>
                         ) : null}
                     </View>
-                    <Badge badge={badge} />
+                    <View testID={`collection_badge_${title}${typeof badge === 'string' ? '_balance_' + badge :  ''}`}>
+                        <Badge badge={badge} />
+                    </View>
```

Comment:
I suggest moving the testID inside the Badge component so as not to change the layout and not to create additional nesting

Reply:
You may need to create a testID prop for the Badge if necessary